extra/planarally.service

```diff
@@ -0,0 +1,15 @@
+[Unit]
+Description=PlanarAlly Server
+After=network.target
+
+[Service]
+Type=simple
+#User www-data is necessary for nginx to work
+User=www-data
+WorkingDirectory=/path/to/PlanarAlly
+ExecStart=/path/to/python3.6+ /path/to/PlanarAlly/server/planarserver.py
```

Comment:
We should add a shebang to planarserver.py as the first line:
```
#!/usr/bin/env python3
```

Reply:
This should also say about virtualenvs:
```systemd
# if you installed PA in a virtualenv, comment the previous line 
# and use the following line:
# ExecStart=/path/to/virtualenv/bin/python /path/to/PlanarAlly/server/planarserver.py
```

Reply:
This is great, thanks O:-) i'll use this on my machine :D i haven't thought of this

Reply:
> We should add a shebang to planarserver.py as the first line and make it executable (chmod +x `planarserver.py`):
> 
> ```
> #!/usr/bin/env python3
> ```

i agree